### PR TITLE
chore(structure): create initial django structure

### LINKED ISSUE #1: Start the Data API app
Build a data API based on [Django-ninja](https://django-ninja.rest-framework.com/).

### PR DESCRIPTION
**Pull request by luabida**
fix #1

### CODE REVIEW

**Comment by luabida** [comment]
@fccoelho what do you think the structure should be?

current:
```
src/
├── manage.py
└── mosqclimate
    ├── asgi.py
    ├── __init__.py
    ├── __pycache__
    │   ├── __init__.cpython-311.pyc
    │   ├── settings.cpython-311.pyc
    │   └── urls.cpython-311.pyc
    ├── settings.py
    ├── urls.py
    └── wsgi.py
   ```

**Comment by fccoelho** [comment]
That's correct, but after that, the tutorial recommends running a `./manage.py startapp <app_name>`

**Comment by fccoelho** [comment]
I don't usually put everything under `src/` I tend to follow the structure recommended by the docs, and created with the `startproject` command.

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+[tool.poetry]
+name = "mosqclimate-data"
```
Comment (fccoelho) [inline pyproject.toml:2]: it's "mosqliamate" without the "c"

File: src/mosqclimate/asgi.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+For more information on this file, see
+https://docs.djangoproject.com/en/3.2/howto/deployment/asgi/
+"""
+
+import os
+
+from django.core.asgi import get_asgi_application
+
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'mosqclimate.settings')
```
Comment (fccoelho) [inline src/mosqclimate/asgi.py:14]: replace "mosqclimate" by "mosqlimate" everywhere

**Review by fccoelho — CHANGES_REQUESTED** [review]

**Comment by luabida** [comment]
> That's correct, but after that, the tutorial recommends running a `./manage.py startapp <app_name>`

I think it would be good to discuss the design of the project, as the apps, the backend integration with postgres, models, containerization etc..

**Comment by fccoelho** [comment]
I suggest we have two apps in the project:

- datastore: this will give access to data on Epigraphhub and Infodengue. Basically a get only API wrapping external databases
- registry: This is a model registry with three tables:
  - Author: table containing the following columns: `name (varchar), email (email), institution (varchar)
  - Model: should contain the following columns: `name (varchar),  author (author id), repository (url), implementation_language (varchar), type (varchar)
  - Predictions: should contain at least the following columns: `model (model id), commit (commit id of the version used), predict_date (datetime), prediction (JSON)

**Comment by luabida** [comment]
This PR is getting too big. I've finished the API for the `registry` app and its endpoints.

How to test it:
- set .env vars (require postgres. currently accepting only external postgres connections)
- run `python src/manage.py runserver`
- access `http://127.0.0.1:8000/api/docs` to test the API requests

**Review by fccoelho — APPROVED** [review]

**Pull request merged**